docs/.vitepress/config.ts

```diff
@@ -477,8 +477,8 @@ function guide(): DefaultTheme.SidebarItem[] {
       link: '/guide/filtering',
     },
     {
-      text: 'Workspace',
-      link: '/guide/workspace',
+      text: 'Test Projects',
+      link: '/guide/projects',
```

Comment:
Would be great if we could redirect old `/guide/workspace` to new `/guide/projects` for a while. Maybe this would do it: https://vitepress.dev/guide/routing#route-rewrites

Reply:
There is a netlify file in this PR already with redirects

Reply:
Great! So many changes I missed it when going through doc updates.